File: CodeWars_CPP_solution/7kyu/Is this a triangle？.cpp
namespace Triangle
{
  bool isTriangle(int a, int b, int c)
  {
    if (a < 0 || b < 0 || c < 0)
      return false;
    long aa, bb, cc;
    aa = a;
    bb = b;
    cc = c;
    if (aa >= bb && aa >= cc)
      return (bb + cc > aa);
    else if (bb >= aa && bb >= cc)
      return (aa + cc > bb);
    else
      return (aa + bb > cc);
  }
};

File: CodeWars_Ruby_solution/7 kyu/Is this a triangle？.rb
def is_triangle(a,b,c)
    p a, b, c
    return false if [a,b,c].min < 0
    if a >= b && a >= c
      return b + c > a
    elsif b >= a && b >= c
      return a + c > b
    else
      return a + b > c
    end
  end